blueprints/checks/concrete_cover/exposure_classes.py

```diff
@@ -0,0 +1,221 @@
+
+
+@total_ordering
+class Chloride(Exposure):
+    """Enum Class which indicates the classification of corrosion induced by chlorides other than by sea water."""
+
+    NA = "Not applicable"
+    XD1 = "XD1"
+    XD2 = "XD2"
+    XD3 = "XD3"
+
+    @staticmethod
+    def description() -> str:
+        """Static method which returns the description of this exposure class.
+
+        Returns
+        -------
+        str
+            description of this exposure class
+        """
+        return "Corrosion induced by chlorides other than by sea water"
+
+
+@total_ordering
+class ChlorideSeawater(Exposure):
+    """Enum Class which indicates the classification of corrosion induced by chlorides from sea water."""
+
+    NA = "Not applicable"
+    XS1 = "XS1"
+    XS2 = "XS2"
+    XS3 = "XS3"
+
+    @staticmethod
+    def description() -> str:
+        """Static method which returns the description of this exposure class.
+
+        Returns
+        -------
+        str
+            description of this exposure class
+        """
+        return "Corrosion induced by chlorides from sea water"
+
+
+@total_ordering
+class Freeze(Exposure):
+    """Enum Class which indicates the classification of freeze/thaw attack with or without de-icing agents."""
+
+    NA = "Not applicable"
+    XF1 = "XF1"
+    XF2 = "XF2"
+    XF3 = "XF3"
+    XF4 = "XF4"
+
+    @staticmethod
+    def description() -> str:
+        """Static method which returns the description of this exposure class.
+
+        Returns
+        -------
+        str
+            description of this exposure class
+        """
+        return "Freeze/thaw attack with or without de-icing agents"
+
+
+@total_ordering
+class Chemical(Exposure):
+    """Enum Class which indicates the classification of chemical attack."""
+
+    NA = "Not applicable"
+    XA1 = "XA1"
+    XA2 = "XA2"
+    XA3 = "XA3"
+
+    @staticmethod
+    def description() -> str:
+        """Static method which returns the description of this exposure class.
+
+        Returns
+        -------
+        str
+            description of this exposure class
+        """
+        return "Chemical attack"
+
+
+class ExposureClasses(NamedTuple):
+    """Named tuple, collects all the different exposure classes of a surface."""
+
+    carbonation: Carbonation = Carbonation("Not applicable")
+    chloride: Chloride = Chloride("Not applicable")
+    chloride_seawater: ChlorideSeawater = ChlorideSeawater("Not applicable")
+    freeze: Freeze = Freeze("Not applicable")
+    chemical: Chemical = Chemical("Not applicable")
+
+    def __str__(self) -> str:
+        """String representation of the ExposureClasses object.
+
+        Returns
+        -------
+        str
+            String representation of the ExposureClasses object
+        """
+        return ", ".join(
+            enum.value
+            for enum in self
+            if enum.value != "Not applicable"  # pylint: disable=not-an-iterable
```

Comment:
can be fix de pylint somehow?

Reply:
good catch, I think this is a bit of legacy code of us, we don't use pylint here ;-)